com.unity.shadergraph/Documentation~/Property-Types.md

```diff
@@ -115,6 +115,8 @@ Displays an HDR color field in the material inspector.
 |:-------------|:------|:------------|
 | Default | Vector 4 | The default value of the [Property](https://docs.unity3d.com/Manual/SL-Properties.html). |
 
+NOTE: Previous versions actually interpreted the color in linear space. The behavior is corrected now, and previously created color properties will continue to use old behavior unless explicilty upgraded through the [Graph Inspector](Internal-Inspector.md). You may use a **Vector 4** property or convert a new **Color** property back to linear space with the [Colorspace Conversion Node](Colorspace-Conversion-Node.md) to mimic old behavior. 
```

Comment:
My suggestion is to move this note higher up (before Default) to make it clearer that this note applies to all color properties. Also, here's my re-write. I'm waiting for Liz to confirm that the info is correct.
```
NOTE: Previous versions actually interpreted the color in linear space. The behavior is corrected now, and previously created color properties will continue to use old behavior unless explicilty upgraded through the [Graph Inspector](Internal-Inspector.md). You may use a **Vector 4** property or convert a new **Color** property back to linear space with the [Colorspace Conversion Node](Colorspace-Conversion-Node.md) to mimic old behavior.
```